vulcanai2/models/basenetwork.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, name, dimensions, config, save_path=None, input_networks=None
         """
         Defines the network object.
         :param name: The name of the network. Used when saving the file.
-        :param dimensions: The dimensions of the network. type: List of tuples
+        :param dimensions: The dimensions of the network. type: List of tuples or int
```

Comment:
since you removed _dimensions, we should rename this to `in_dims` for consistency variable naming